### ISSUE TITLE
Compilation error with `USE_SimilarityTransformElastix`

### ISSUE THREAD

**Issue by hmaarrfk** [SuperElastix/elastix#72]
When compiled with `USE_SimilarityTransformElastix`

```
/home/mark/git/elastix/Common/Transforms/itkAdvancedSimilarity3DTransform.hxx:103:37:   required from ‘void itk::AdvancedSimilarity3DTransform<TScalarType>::SetMatrix(const MatrixType&) [with TScalarType = double; itk::AdvancedSimilarity3DTransform<TScalarType>::MatrixType = itk::Matrix<double, 3, 3>]’
/home/mark/git/elastix/Common/Transforms/itkAdvancedSimilarity3DTransform.hxx:83:1:   required from here
/usr/include/vxl/core/vnl/vnl_math.h:175:21: error: cannot bind non-const lvalue reference of type ‘double&’ to an rvalue of type ‘double’
     return std::cbrt(std::forward<TArg>(arg));
```

**Comment by hmaarrfk** [comment]
Gah, sorry, this seems to be an issue with `vxl` maybe? I installed the 2018 06 18 version.

**Comment by kaspermarstal** [comment]
What version of ITK are you using?

**Comment by mstaring** [comment]
We are using the version shipped with ITK 4.13.x. We are not yet moving to ITK 5, and newer vxl versions embedded in there. Feel free to reopen if the problem also occurs with version 4.13.
